app/models/user.rb

```diff
@@ -4,6 +4,8 @@ class User < ActiveRecord::Base
   has_one :budget
 
   validates :name, :email, presence: true, uniqueness: true
+  validates :email, confirmation: true
+  validates :password, length: { in: 6..20 }
```

Comment:
Does password validation belong in this PR?

Reply:
If limiting password length, I would limit it at a higher number, for example 64. 20 is not secure enough.

You might want to add other validation rules to password, for example using letters and numbers. Maybe in another PR?

Reply:
Done in sha: 1159cb8
Will look at another PR to set stronger validation rules for password.